<launch>

  <node pkg="rollout" name="rollout_actor" type="rollout_actor.py" output="screen"/>
  <node pkg="rollout" name="rollout_recorder" type="rollout_recorder.py" output="screen"/>
  <node pkg="hand_control" name="keyboard_control" type="keyboard_control.py" output="screen"/>
  <node pkg="rollout" name="rollout_restart" type="rollout_reset.py" output="screen"/>

</launch>
